content/kubermatic/main/installation/offline-mode/_index.en.md

```diff
@@ -128,6 +128,62 @@ spec:
     - quay.io/kubermatic/kubelb-manager-ee:v1.1.0
 ```
 
+## Mirroring Binaries  
+The `kubermatic-installer mirror-binaries` command is designed to **mirror and host essential binaries** required by the Operating System Profiles for provisioning user clusters in **offline/airgapped environments**. This includes critical components like:  
+- **Kubernetes binaries**: `kubeadm`, `kubelet`, `kubectl`  
+- **CNI plugins** (e.g., Calico, Cilium)  
```

Comment:
The plugins in the CNI plugins are not Calico and Cilium; https://github.com/containernetworking/plugins?tab=readme-ov-file#plugins-supplied